setup.py

```diff
@@ -66,9 +66,9 @@
         'License :: OSI Approved :: MIT License',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
```

Comment:
Can add 3.8 here.

Reply:
We should also change the testing matrix in the travis and appveyor so these versions get tested.

Reply:
I don't see an appveyor.yml file?

Reply:
My bad, I think that's only ripser.py that has appveyor setup.

Reply:
K i updated